bfscripts.spec.rpkg

```diff
@@ -88,14 +88,14 @@ systemctl daemon-reload
 
 %files
 %license LICENSE
-%defattr(-,root,root)
 /usr/bin/bf*
 /usr/bin/mlx-mkbfb
-%{_mandir}/man1/*
-%{_mandir}/man8/*
-%{_unitdir}/bfvcheck.service
+%attr(644, root, root) %{_mandir}/man1/*
+%attr(644, root, root) %{_mandir}/man8/*
+%attr(644, root, root) %{_unitdir}/bfvcheck.service
 
 %doc README.md
 
 %changelog
-{{{ git_dir_changelog }}}
+* Wed Jul 1 2020 Spencer Lingard <spencer@nvidia.com> - 3.0.0~beta1-1
```

Comment:
Just a question, why not use '-' instead of '~' here?  Or the script tools will replace the '~' to '-'?

Reply:
We need to use a `~` character because of fedora's guidelines for prerelease versioning: https://docs.fedoraproject.org/en-US/packaging-guidelines/Versioning/#_versioning_prereleases_with_tilde